### ISSUE TITLE
can't change default font size

### ISSUE THREAD

**Issue by tisdall** [froala/wysiwyg-editor#3691]
The following help document seems to be wrong:  https://wysiwyg-editor.froala.help/hc/en-us/articles/115000817105-How-can-I-set-a-default-font-style-

There's CSS styling that explicitly sets the font-size to 14px and binds closer then `.fr-view`:  So, setting a style on `.fr-view` will work on the displayed content, but not in the editor due to the following:

https://github.com/froala/wysiwyg-editor/blob/v3.0.5/css/froala_editor.css#L36

```
.fr-box.fr-basic .fr-element {
  font-size: 14px;
}
```

**Comment by jmiserez** [comment]
Yes, in fact the rule needs to be quite a bit more specific. I'm using ".myfield.fr-box.fr-basic .fr-view.fr-element" but anything with spec >= 0-3-0 should work (see spec calculator here: https://specificity.keegan.st)

**Comment by casualuser** [comment]
Hello,

Please try extended css paths like this:
```
.fr-box.fr-basic .fr-element.fr-view {
  font-family: "Times New Roman", Georgia, Serif;
  font-size: 18px;
  color: #444444;
}
```

We also updated that help article - https://wysiwyg-editor.froala.help/hc/en-us/articles/115000817105-How-can-I-set-a-default-font-style-

**Comment by jondavidpass** [comment]
Also experiencing the same issue even with a more specific selector. Also setting `fontSizeDefaultSelection` which doesn't appear to make any difference either.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
